ui/widgets/nuxeo-date-picker.js

```diff
@@ -269,12 +268,6 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
       }
       this._preventInputUpdate = false;
     }
-
-    _firstDayOfWeekChanged() {
-      if (this.firstDayOfWeek != null) {
-        this.$.date.set('i18n.firstDayOfWeek', this.firstDayOfWeek);
```

Comment:
Do we need to removed this? We will lose the ability to change the first day of the week after the element was stamped. It would make sense in case it didn't actually work.

Reply:
It was not working, in fact. But I wasn't able to track why.